requirements/test.txt

```diff
@@ -76,7 +77,7 @@ pymongo==3.10.1           # via edx-opaque-keys
 pyparsing==2.4.7          # via packaging
 pytest-cov==2.10.0        # via -r requirements/test.in
 pytest-django==3.9.0      # via -r requirements/test.in
-pytest==5.4.3             # via pytest-cov, pytest-django
```

Comment:
this gave me lots of trouble in edx-enterprise, will be good to confirm if builds pass with this, if so all good! I think edx-enterprise upgrade of this is still open